### ISSUE TITLE
[Bug][Tabstrip] Tabstrip incorrectly sets aria-hidden to rendered tab when keepTabsMounted

### ISSUE THREAD

**Issue by zoran995** [telerik/kendo-react#1988]
## I'm submitting a...
* Regression report
* Bug report

## Current behavior
- kendo react sets `aria-hidden: true` to rendered tab element when keepTabsMounted is set to true

## Expected behavior
- kendo should set `aria-hidden: true` to non-visible tabs
- kendo should set `aria-hidden: false` to currently selected tab content

## Minimal reproduction of the problem with instructions
1. Open stackblitz https://stackblitz.com/~/github.com/zoran995/kendo-tabstrip-repro, you can find source on github https://github.com/zoran995/kendo-tabstrip-repro
2. check the HTML of rendered app and find the content with class `k-tabstrip-content k-active`
3. notice it has `aria-hidden: true` set, but it is rendered on screen
4. from terminal run `npm run test` notice that one test is failing (it is using the `keepTabsMounted={true}`)

<img width="681" alt="image" src="https://github.com/telerik/kendo-react/assets/19208948/fb54f86e-861f-46cb-8395-74edae110396">

## What is the motivation or use case for changing the behavior?
This breaks the ability to write unit tests using accessibility features of the react testing library, and it worked before upgrading to v7.

**Comment by PekoPPT** [comment]
Hi, @zoran995. Thank you for your report. The issue is a known one and is also discussed in this thread - https://github.com/telerik/kendo-react/issues/931. We will do our best to prioritize the task and implement it in the current quarter. The TabStrip should have the expected behavior with our May release, at the latest. 

Currently, I cannot say exactly when the bug will be fixed, but once this is done, we will write here. 
Please let me know if you have additional questions related to the issue.

**Comment by jbennie** [comment]
in the generated code the @HostBinding Selectors has a typo. 

In my debug panel I get the following 

 ```
 k.\u0275cmp = e.VBU({
                type: k,
                selectors: [["", "kendoTabStripTab", ""]],
                hostVars: 13,
                hostBindings: function(S, P) {
                    2 & S && (e.BMQ("aria-selected", P.activeClass)("aria-disabled]", P.disabledClass)("tabindex", P.tabIndex),
                    e.AVh("k-item", P.hostClasses)("k-tabstrip-item", P.hostClasses)("k-active", P.activeClass)("k-disabled", P.disabledClass)("k-focus", P.focusedClass))
                },
                inputs: {
                    tab: "tab",
                    index: "index",
                    tabStripClosable: "tabStripClosable",
                    tabStripCloseIcon: "tabStripCloseIcon",
                    customTabstripCloseIcon: "customTabstripCloseIcon",
                    closeSVGIcon: "closeSVGIcon"
                },
                outputs: {
                    tabClose: "tabClose"
                },
```
the code on this line 


`  2 & S && (e.BMQ("aria-selected", P.activeClass)("aria-disabled]", P.disabledClass)("tabindex", P.tabIndex),`

needs to be 

`  2 & S && (e.BMQ("aria-selected", P.activeClass)("aria-disabled", P.disabledClass)("tabindex", P.tabIndex),`

there is an extra `]`   which implies is a bug with an @hostBinding selector. or the html its binding to is malformed.  

Going over to git hub and scanning the code 

`Line 700 of https://github.com/telerik/kendo-ui-core/blob/master/src/kendo.tabstrip.js reads. `

` referenceContent = that.element.find("[id='" + referenceTab.attr(ARIA_CONTROLS) + "']");`

should probably be 

` referenceContent = that.element.find("[id]='" + referenceTab.attr(ARIA_CONTROLS) + "'");`

Just a guess - I can't currently get into debugging it ;)  what ever is being found and replaced in the subsequent code is introducing the broken html.

**Comment by jbennie** [comment]
NodeModeules ...... File fesm2015 > progress-kendo-angular-layout.mjs ... edit the "attr.aria-disabled]" to "attr.aria-disabled" 
LN 3447, col 578 
`TabComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "13.3.12", type: TabComponent, selector: "[kendoTabStripTab]", inputs: { tab: "tab", index: "index", tabStripClosable: "tabStripClosable", tabStripCloseIcon: "tabStripCloseIcon", customTabstripCloseIcon: "customTabstripCloseIcon", closeSVGIcon: "closeSVGIcon" }, outputs: { tabClose: "tabClose" }, host: { properties: { "class.k-item": "this.hostClasses", "class.k-tabstrip-item": "this.hostClasses", "attr.aria-selected": "this.activeClass", "class.k-active": "this.activeClass", "attr.aria-disabled": "this.disabledClass", "class.k-disabled": "this.disabledClass", "class.k-focus": "this.focusedClass", "attr.tabindex": "this.tabIndex" } }, ngImport: i0, template: ``
 
fesm2020. Ln 3437 col 578
esm2020 Ln 75 col 578

**Comment by PekoPPT** [comment]
Hi, @jbennie. 
The shared details are related to the **Kendo UI for Angular** suite. 

KendoReact and Kendo UI for Angular suites don't use the same codebase and the experienced issue should be addressed here: https://github.com/telerik/kendo-angular/issues.

Please submit a new issue about the scenario you shared and our colleagues from the Angular team will handle the issue.

**Comment by PekoPPT** [comment]
Hi, @zoran995. 
Based on your report, in version 3.4.0 of the KendoReact components, the TabStrip will have a new property called **renderAllContent**. When set to **true**, the property will force the initial rendering of all tabs, part of the TabStrip component.
The new property is already available in a development version - https://www.npmjs.com/package/@progress/kendo-react-common/v/7.4.0-develop.7 - and you can test it before the official release. 

[Here is a StackBlitz example](https://stackblitz.com/edit/react-pegxag-kwhqws?file=app%2Fmain.tsx) that uses the mentioned above version **7.4.0-develop.7**.

Using the discussed prop, we can configure the TabStrip, based on the business requirements of each project.
